README.md

```diff
@@ -522,6 +522,18 @@ System variables provide a pre-defined set of variables that can be used in any
   `<domain|tenantId>`: Optional. Domain or tenant id for the directory to sign in to. Default: Pick a directory from a drop-down or press `Esc` to use the home directory (`common` for Microsoft Account).
 
   `aud:<domain|tenantId>`: Optional. Target Azure AD app id (aka client id) or domain the token should be created for (aka audience or resource). Default: Domain of the REST endpoint.
+* `{{$aadV2Token [new] [appOnly ][scopes:<scope[,]>] [tenantid:<domain|tenantId>] [clientid:<clientId>]}}`: Add an Azure Active Directory token based on the following options (must be specified in order):
```

Comment:
Could you please add some description about when/why/what is the AAD V2 Auth, especially what's the difference between the v1 and v2?

Reply:
Will do